gui/slick/views/comingEpisodes.mako

```diff
@@ -23,6 +23,9 @@
 <%namespace file="/inc_defs.mako" import="renderQualityPill"/>
 <h1 class="header">${header}</h1>
 <div class="h2footer pull-right">
+% if layout != 'poster':
+    ## <button id="popover" type="button" class="btn btn-inline">Select Columns <b class="caret"></b></button> <!-- Not sure why this doesn't work -->
```

Comment:
For some reason if we try to add this button to comingEpisodes and then the popover js runs we get a warning about a missing config. "Uncaught TypeError: Cannot read property 'config' of undefined".